### ISSUE TITLE
Package as a gem

### ISSUE THREAD

**Issue by rockyallen** [asciidoctor/docbookrx#65]
Can it be packaged for "gem install" please? 
Reasons:
I can't get the install to work on Ubuntu (Nokogiri again)
I need to install/run it offline

**Comment by mojavelinux** [comment]
Yes. The reason I've been holding off is because I have a new code design that I'm waiting to apply that will make it more package friendly (and a lot easier to develop). When I bring over that code, I'll also set up the gem release.

**Comment by NicolBolas** [comment]
Is this project still live? It's been almost a year, and there's no new code design and no gem support. And Nokogiri apparently doesn't work for Ruby 2.7.0.0.

**Comment by mojavelinux** [comment]
I haven't had a chance to come around and rework it yet. So the situation is still the same.
